generative_data_prep/data_prep/pipeline.py

```diff
@@ -433,6 +438,17 @@ def pipeline_main(  # noqa: C901
         dev_ratio,
         test_ratio,
     )
+
+    num_splits_greater_lines = True
+    with open(input_file_path, "r") as input_file:
+        for i, line in enumerate(input_file):
+            if i > num_splits:
+                num_splits_greater_lines = True
+                break
+    if not num_splits_greater_lines:
+        raise ValueError(
+            "The number of total splits exceeds the number of data entries. Please reduce the number of splits."
```

Comment:
nit: update message from  
"The number of total splits exceeds the number of data entries. Please reduce the number of splits." 
to 
"The number of total splits exceeds the number of lines in the input path jsonl file. Please reduce the number of splits, or increase the number of lines in the dataset."